README.md

```diff
@@ -46,6 +59,10 @@ Keep in mind that now it is not possible to select the content with
 mouse, so apply the `cursor: default;` CSS style to prevent confusing
 the users (or even `cursor: grab;` in case the content is not a text).
 
+You can use `zoom-in(number)`,`zoom-out(number)` to zoom in and zoom out a `dragsimg`.  
```

Comment:
Shoud I always add `id="dragsimg"` to the scrolled image, or is it just an example?